.github/ISSUE_TEMPLATE/--onboarding.yaml

```diff
@@ -48,7 +48,7 @@ body:
     id: type
     attributes:
       label: Image type
-      description: Is this a ROCK or a Dockerfile-based image?
+      description: description of the rock image
```

Comment:
this whole field can actually be removed no? if we don't support docker images anymore, there's no option -> it must be a rock